lib/cloudinary/utils.rb

```diff
@@ -996,7 +997,22 @@ def self.generate_auth_token(options)
     Cloudinary::AuthToken.generate options
 
   end
-  
+
+  # Creates and returns a URL that when invoked creates an archive of a folder.
+  #
+  # @param [Object] folder_path Full path (from the root) of the folder to download.
+  # @param [Hash] options       Additional options.
+  #
+  # @return [String]
+  def self.download_folder(folder_path, options = {})
+    options[:resource_type] = options[:resource_type] || "all"
+    options[:prefixes] = folder_path
+
+    cloudinary_params = Cloudinary::Utils.sign_request(Cloudinary::Utils.archive_params(options.merge(:mode => "download")), options)
```

Comment:
You can call `download_archive_url` directly, in a similar way it is used in `download_zip_url`